adev/shared-docs/services/search.service.ts

```diff
@@ -233,3 +236,24 @@ function wait(ms: number, signal: AbortSignal): Promise<void> {
     );
   });
 }
+
+function extractPackageNameFromUrl(url: string): string | null {
+  const apiMarker = '/api/';
+  const apiIndex = url.indexOf(apiMarker);
+
+  if (apiIndex === -1) {
+    return null;
+  }
+
+  const path = url.substring(apiIndex + apiMarker.length);
+  const parts = path.split('/');
+
+  // The last part is the symbol name, we don't need it.
+  const packageParts = parts.slice(0, -1);
+
+  if (packageParts.length === 0) {
+    return null;
+  }
+
+  return `From <code>@angular/${packageParts.join('/')}</code>`;
```

Comment:
Would this work instead?

```suggestion
  const extractedSegment = url.match(/\/api\/(.*)#?/);
  if (extractedSegment == null) {
    return null;
  }
  return `From <code>@angular/${extractedSegment[1]}</code>`;
```

Reply:
We need to drop the symbol name from the url, but yes. Updated.